### ISSUE TITLE
Incorrect ARIA values?

### ISSUE THREAD

**Issue by norfali** [leongersen/noUiSlider#889]
According to ARIA guidelines in a Slider (in a range of 0 to 20):

- Lower handle should always have `aria-valuemin` of 0
- Upper handle should always have `aria-valuemax` of 20.
- Lower handle should have `aria-valuemax` based on Upper handle value.
- Upper handle should have `aria-valuemin` based on Lower handle value.
- `aria-valuenow` should match the exact position of the associated handle.

It appears that the numbers are percentage based (or hardcoded values of 0 and 100?).

See https://codepen.io/norfali/pen/mKWoxp for demonstration of the defect and the correct implementation: https://www.w3.org/TR/wai-aria-practices/examples/slider/multithumb-slider.html

**Comment by codinronan** [comment]
Just ran into this too. Not a *huge* deal for us yet, but one of my goals on this particular project was near 100% accessibility :) Ideally really you'd be able to format these custom - but even just using the input range as above would be more correct than what it's doing now..

**Comment by leongersen** [comment]
I just released noUiSlider 12.0.0 that resolves this issue. Thank you very much for reporting!
